app/serializers/assigned_group_serializer.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class AssignedGroupSerializer < ApplicationSerializer
+  attributes :id, :name, :assign_icon, :assign_path
+
+  def assign_icon
+    "group-plus"
+  end
+
+  def assign_path
+    "/g/#{object.name}/assigned/everyone"
```

Comment:
This isn't new to this PR, but it doesn't look like this path is being used anywhere...

Reply:
This shouldn't hold merging the PR, but it does look like a separate regression.

Reply:
Will keep the attribute then. A lot has changed since the last time I worked on this plugin, so not sure how it should be used.